### PR TITLE
Check minimal k8s version before installation

### LINKED ISSUE #1195: Add helm check for the minimal k8s version requirement

### PR DESCRIPTION
**Pull request by sumo-drosiek**
###### Description

Fixes #1195 

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: deploy/helm/sumologic/templates/_helpers.tpl [hunk 1]
```diff
@@ -925,3 +925,15 @@ Example:
 {{- $_ := set $local "first" false -}}
 {{- end -}}
 {{- end -}}
+
+
+{{/*
+Returns kubernetes version minor as integer (without additional chars like +)
```
Comment (sumo-drosiek) [inline deploy/helm/sumologic/templates/_helpers.tpl:931]:
```suggestion
Returns kubernetes minor version as integer (without additional chars like +)
```

File: deploy/helm/sumologic/templates/_helpers.tpl [hunk 2]
```diff
@@ -925,3 +925,15 @@ Example:
...
+
+Example:
+
+{{ include "kubernetes.version.minor" . }}
```
Comment (sumo-drosiek) [inline deploy/helm/sumologic/templates/_helpers.tpl:935]:
```suggestion
{{ include "kubernetes.minor" . }}
```

**Review by perk-sumo — COMMENTED** [review]
I'm overall good but please make it 2 commits (one for .gitignore and another for the changes) or split to 2 PRs.

**Review by astencel-sumo — APPROVED** [review]

**Comment by sumo-drosiek** [comment]
@perk done

**Review by astencel-sumo — APPROVED** [review]

**Pull request merged**